docs/v8-migration.md

```diff
@@ -11,6 +11,21 @@ If you are working directly with Babel's API (because, for example, you maintain
 
 > If you are upgrading from Babel 6, please check the [Babel 7 migration guide](./v7-migration.md) first.
 
+## Getting ready {#getting-ready}
+
+Before migrating to Babel 8, we recommend that you update your Babel 7 configuration to align it with Babel 8's new defaults. Once your build process is working again with Babel 7, you can then upgrade to Babel 8.
+
+You should read this full document to understand what options you need to change and what plugins you will need to remove, but here are the most important steps that you should take to modernize your config:
+- If you are not using a `.browserslistrc` file, define a top-level [`targets`](https://babeljs.io/docs/options#targets) option. The Babel 7 default is `targets: ">= 0%"` (i.e. all browsers), while the Babel 8 default is [`targets: "defaults"`](https://browsersl.ist/#q=defaults). If you are using [`@babel/preset-env`'s `targets`](https://babeljs.io/docs/babel-preset-env#targets) option, you can copy its value to the top-level configuration (next to the `presets` option).
+- If you are using `@babel/preset-env`, enable its [`bugfixes`](https://babeljs.io/docs/babel-preset-env#bugfixes) option.
+- If you are using `@babel/preset-env`'s `loose` or `spec` options, [migrate to `assumptions`](https://babeljs.io/docs/assumptions#migrating-from-babelpreset-envs-loose-and-spec-modes).
+- If you are using `@babel/preset-react` or `@babel/plugin-transform-react-jsx`, explicitly set their [`runtime`](https://babeljs.io/docs/babel-preset-react#runtime) option (Babel 7 defaults to `"classic"`, while Babel 8 to `"automatic"`), and if you choose to keep using the classic runtime set the [`useSpread`](https://babeljs.io/docs/babel-plugin-transform-react-jsx#usespread) option to `true`.
+- If you are using the TypeScript or Flow presets, replace the `isTSX` and `allExtensions` options with [`ignoreExtensions`](#babel-preset-typescript).
+- If you are transforming TypeScript or Flow, set the `allowDeclareFields` option to true (https://babeljs.io/docs/babel-preset-typescript#allowdeclarefields).
+- If you are using [`@babel/plugin-transform-runtime`'s `corejs`](https://babeljs.io/docs/babel-plugin-transform-runtime#corejsu) or [`@babel/preset-env`'s `useBuiltIns`](https://babeljs.io/docs/babel-preset-env#usebuiltins) options, remove it and use [`babel-plugin-polyfill-corejs3`](https://github.com/babel/babel-polyfills/tree/main/packages/babel-plugin-polyfill-corejs3) instead.
+- If you are using `@babel/plugin-proposal-decorators`, set its [`version`](https://next.babeljs.io/docs/babel-plugin-proposal-decorators/#version) option to `legacy` or `2023-11`.
```

Comment:
Can you use the relative markdown links? In the future if `babeljs.io` points to the Babel 8 docs, these links should still work for `v7.babeljs.io`.

Reply:
Actually, maybe these links should just point to v7? Since they are about what to do _before_ upgrading to v8.

Reply:
The reason I prefer relative markdown links, such as `./babel-plugin-transform-runtime.md#corejsu` is that docusaurus can check whether this link is still valid when we change the markdown structure. Since the links here are mostly option documents, how about a relative link to current docs and then fallback to the v7 doc if such option has been removed?